extensions/amp-story/1.0/test/validator-amp-story-amp-render.html

```diff
@@ -0,0 +1,98 @@
+  See the License for the specific language governing permissions and
+  limitations under the license.
+-->
+<!--
+  Test Description:
+  Tests for the amp-render functionality inside amp-story.
+-->
+<!DOCTYPE html>
+<html amp lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script async custom-element="amp-story"
+      src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+    <script async custom-element="amp-render"
+      src="https://cdn.ampproject.org/v0/amp-render-1.0.js"></script>
+    <script async custom-template="amp-mustache"
+      src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"></script>
+    <script async custom-element="amp-bind"
+      src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+    <title>amp-render</title>
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1"/>
+    <link rel="canonical" href="index.html" />
+    <style amp-boilerplate>
+      body{
+        -webkit-animation:-amp-start 8s steps(1, end) 0s 1 normal both;
+        -moz-animation:-amp-start 8s steps(1, end) 0s 1 normal both;
+        -ms-animation:-amp-start 8s steps(1, end) 0s 1 normal both;
+        animation:-amp-start 8s steps(1, end) 0s 1 normal both
+      }
+      @-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}
+      @-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}
+      @-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}
+      @-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}
+      @keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}
+    </style>
+    <noscript>
+      <style amp-boilerplate>
+        body{
+          -webkit-animation:none;
+          -moz-animation:none;
+          -ms-animation:none;
+          animation:none
+        }
+      </style>
+    </noscript>
+  </head>
+
+  <body>
+    <amp-story standalone title="My Story" publisher="Me"
+      publisher-logo-src="http://me.com/logo.png"
+      poster-portrait-src="http://me.com/poster.jpg">
+      <amp-story-page id="page-rendering">
+        <amp-story-grid-layer template="vertical">
+          <amp-render
+            layout="fixed-height"
+            height="100"
+            src="/examples/images.json"
+            binding="no">
+            <template type="amp-mustache">
+              <div><a href="{{url}}">{{title}}</a></div>
+            </template>
+          </amp-render>
+          <div placeholder>Loading ...</div>
+          <div fallback>Failed to load data.</div>
+        </amp-story-grid-layer>
+      </amp-story-page>
+
+      <amp-story-page id="attachment-rendering">
+        <amp-story-grid-layer template="fill">
+          <amp-img src="/examples/" width="900" height="1600">
+          <template type="amp-mustache" id="my-template">
```

Comment:
Nit: plz move the template so it's by its amp-render component
Also re: working example

Reply:
I actually get the following error whenever I try to make the template adjacent to its amp-render component:
`The tag 'template' may not appear as a descendant of tag 'amp-story-page-attachment'`

So it seems that `amp-story-grid-layer` is the closest the template can go

Reply:
I expect publishers to run into this issue as well, we should allow for the `<template` tag too

Reply:
Sounds good, done